Terminal.Gui/Views/GraphView/Annotations.cs

```diff
@@ -155,21 +170,25 @@ public LegendAnnotation (Rect legendBounds)
 		/// <param name="graph"></param>
 		public void Render (GraphView graph)
 		{
-			if (Border) {
-				graph.Border.DrawFrame (Bounds, true);
+			if (!_legend.IsInitialized) {
+				_legend.ColorScheme = new ColorScheme () { Normal = Application.Driver.GetAttribute () };
+				graph.Add (_legend);
 			}
```

Comment:
@tig here is the adding the _legend as subview of the graph. If you don't want as view and you want as before I have no problem to change it. It only was to prove the border work for the _legend.